README.md

```diff
@@ -1,36 +1,36 @@
 # Magic-settings
 
-## Установка
+## Installation
 
 ```bash
 $ pip install magic-settings
 ```
 
-Для использования настроек из `yaml` файла
+Using settings from `yaml` file
 
 ```bash
 $ pip install magic-settings[yaml]
 ```
 
-## Инициализация
+## Initialization
 
-### Объявление класса настроек проекта
+### Project settings class declaration
```

Comment:
```suggestion
### Project settings class declaration

```

[MD022 - Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md022)